aliases

```diff
@@ -9,6 +9,7 @@ alias v="$VISUAL"
 alias b="bundle"
 
 # Rails
+alias extension_server="export RUN_SERVER_FOR_CHROME_EXTENSION=true; rake assets:precompile && thin start --ssl -p 7000; export RUN_SERVER_FOR_CHROME_EXTENSION=false;"
```

Comment:
@arktisklada going to merge this in for now as it works. We can always update the alias implementation if you find a better solution